Dungeon Scaler/Assets/Scripts/Player_Scripts/Player.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour {
+
+    [SerializeField] private int maxHealth = 5;
+    [SerializeField] private int currentHealth;
+
+    [SerializeField] private HealthBar healthBar;
+
+    private void Start() {
+        currentHealth = maxHealth;
+        healthBar.SetMaxHealth(maxHealth);
+    }
+
+    private void Update() {
+        if ( Input.GetKeyDown(KeyCode.Space) ) {
+            TakeDamage(1);
```

Comment:
It's likely that you'll have different damage amounts eventually. I'd recommend creating an enum that represents base damage amounts. Might not have to be done now if you feel like it is unnecessary at this stage -- just trying to think ahead a little.

Reply:
I made an enum named Damage that will hold the damage dealt to the player and all of the different damages dealt by the weapons the player will be holding. This may be moved into a weapon class later.
`
public enum Damage{
        PlayerHurt = 1, //damage dealt to player when hit by enemy
        SwordDamage = 1 //damage dealt by sword weapon
    }
...
if ( Input.GetKeyDown(KeyCode.Space) ) {
            TakeDamage();
}
...
private void TakeDamage() {
        currentHealth -= (int)Damage.PlayerHurt;
        healthBar.SetHealth(currentHealth);
    }
`